examples/psf_wf_movie.py

```diff
@@ -0,0 +1,258 @@
+    JT_alts = [0.0, 2.58, 5.16, 7.73, 12.89, 15.46]  # km
+    JT_weights = [0.652, 0.172, 0.055, 0.025, 0.074, 0.022]  # weight
+    JT_interp = galsim.LookupTable(JT_alts, JT_weights, interpolant='linear')
+
+    alts = 15.46*np.arange(args.nlayers)/(args.nlayers-1)  # uniformly spaced altitudes in km.
+    weights = JT_interp(alts)  # interpolate the weights
+    weights /= sum(weights)  # and renormalize
+
+    # Each layer can have its own turbulence strength (roughly inversely proportional to the Fried
+    # parameter r0), wind speed, wind direction, altitude, and even size and scale (though note that
+    # the size of each screen is actually made infinite by "wrapping" the edges of the screen.)  The
+    # galsim.Atmosphere helper function is useful for constructing this list, and requires lists of
+    # parameters for the different layers.
+
+    spd = []  # Wind speed in m/s
+    dirn = [] # Wind direction in radians
+    r0_500 = [] # Fried parameter in m at a wavelength of 500 nm.
+    for i in xrange(args.nlayers):
+        spd.append(u()*20.0)  # Use a random speed between 0 and 20 m/s for each layer
+        dirn.append(u()*360*galsim.degrees)  # And an isotropically distributed wind direction.
+        r0_500.append(args.r0_500*weights[i]**(-3./5))
+        print ("Adding layer at altitude {:5.2f} km with velocity ({:5.2f}, {:5.2f}) m/s, "
+               "and r0_500 {:5.3f} m."
+               .format(alts[i], spd[i]*dirn[i].cos(), spd[i]*dirn[i].sin(), r0_500[i]))
+
+    # Additionally, we set the screen temporal evolution `time_step`, and the screen size and scale.
+    atm = galsim.Atmosphere(r0_500=r0_500, speed=spd, direction=dirn, altitude=alts, rng=rng,
+                            time_step=args.time_step, screen_size=args.screen_size,
+                            screen_scale=args.screen_scale)
+    # `atm` is now an instance of a galsim.PhaseScreenList object.
+
+    # Place to store the cumulative PSF image if args.accumulate is set.
+    psf_img_sum = galsim.ImageD(args.psf_nx, args.psf_nx, scale=args.psf_scale)
+
+    # Field angle (angle on the sky wrt the telescope boresight) at which to compute the PSF.
+    theta = (args.x*galsim.arcmin, args.y*galsim.arcmin)
+
+    # Construct an Aperture object for computing the PSF.  The Aperture object describes the
+    # illumination pattern of the telescope pupil, and chooses good sampling size and resolution
+    # for representing this pattern as an array.
+    aper = galsim.Aperture(diam=args.diam, lam=args.lam, obscuration=args.obscuration,
+                           nstruts=args.nstruts, strut_thick=args.strut_thick,
+                           strut_angle=args.strut_angle*galsim.degrees,
+                           screen_list=atm, pad_factor=args.pad_factor,
+                           oversampling=args.oversampling)
+
+    # Code to setup the Matplotlib animation.
+    FFMpegWriter = anim.writers['ffmpeg']
```

Comment:
And this is matplotlib 1.1.0, sorry.  (I spent hours trying to upgrade to 1.5.1 a while back... should really fix that.  But for now I'm stuck.)  I'm not proposing that you support super old matplotlib versions, but perhaps check the version when you import and fail at that point if the version is too old?